.github/workflows/ci.yml

```diff
@@ -1,124 +1,134 @@
 name: CI
 
+env:
+    APPLICATION_ENV: 'development'
+    APPLICATION_STORE: 'DE'
+    PROJECT: 'Payone'
+    DATABASE_VERSION: 10.2
+    DATABASE_HOST: 127.0.0.1
+    DATABASE_PORT: 3306
+    DATABASE_NAME: eu-docker
+    DATABASE_USERNAME: root
+    DATABASE_PASSWORD: secret
+    DATABASE_ROOT_PASSWORD: secret
+    DATABASE_ALLOW_EMPTY_PASSWORD: false
+    DATABASE_CHARACTER_SET: utf8
+    DATABASE_COLLATE: utf8_general_ci
+    MYSQL_ALLOW_EMPTY_PASSWORD: false
+    MYSQL_ROOT_PASSWORD: secret
+    MYSQL_DATABASE: eu-docker
+
 on:
     pull_request:
     push:
         branches:
             - master
     workflow_dispatch:
 
-env:
-    SPRYKER_TESTING_ENABLED: 1
-    APPLICATION_STORE: DE
-    POSTGRES_PORT: 5433
-    DB_USER: postgres
-    DB_NAME: DE_test_zed
-    DB_PASSWORD: pwd
-    MODULE_DIR: module
-    SHOP_DIR: current
-    MODULE_NAME: payone
-
 jobs:
-    test_pgsql:
-        name: "Test / PostgreSQL"
+    ci:
+        name: Payone
         runs-on: ubuntu-18.04
+
         strategy:
             fail-fast: false
             matrix:
-                php: [ '7.3', ]
-                product: [ 'suite' ]
-                composer_preference: ['', '--prefer-lowest']
-        services:
-            rabbitmq:
-                image: rabbitmq
-                ports:
-                    - 5672:5672
-            redis:
-                image: redis
-                ports:
-                    - 6379:6379
-            postgres:
-                image: postgres:12
-                env:
-                    POSTGRES_PASSWORD: ${{ env.DB_PASSWORD }}
-                    DB_USER: ${{ env.DB_USER }}
-                    DB_NAME: ${{ env.DB_NAME }}
-                ports:
-                    - 5433:5432
-                options: >-
-                    --health-cmd pg_isready
-                    --health-interval 10s
-                    --health-timeout 5s
-                    --health-retries 5
-        env:
-            PRODUCT_NAME: ${{ matrix.product }}
-            APPLICATION_ENV: ci.pgsql
+                php-versions:
+                    - '7.4'
+                    - '8.0'
```

Comment:
8.1 now, see https://github.com/spryker-eco/payone/pull/77

Reply:
Sounds good, I'll change.

Reply:
Error in vendor/propel/propel/src/Propel/Generator/Builder/Om/ClassTools.php for 8.1
strrpos(): Passing null to parameter #1 ($haystack) of type string is deprecated
Propel module isn't ready for 8.1